os/include/tinyara/wifi_csi/wifi_csi_common.h

```diff
@@ -0,0 +1,84 @@
+/****************************************************************************
+ *
+ * Copyright 2024 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+#ifndef __INCLUDE_TINYARA_WIFI_CSI_COMMON_H
```

Comment:
Are you able to distinguish `wifi_csi.h` and `wifi_csi_common.h` in usage? How about renaming them to know what it is from the name?

Reply:
Changed.